irohad/ametsuchi/impl/postgres_block_query.cpp

```diff
@@ -242,5 +242,19 @@ namespace iroha {
       return block_store_.last_id();
     }
 
+    expected::Result<BlockQuery::wBlock, std::string>
+    PostgresBlockQuery::getTopBlock() {
+      auto block =
+          block_store_.get(block_store_.last_id()) | [](const auto &bytes) {
+            return shared_model::converters::protobuf::jsonToModel<
```

Comment:
We have protobuf leaking here. We need a task to implement converters which return interface objects. Could you add a task for that?

Reply:
These tasks should be sufficient:
https://soramitsu.atlassian.net/browse/IR-937 Rework shared_model interface with getTransport method with default type
https://soramitsu.atlassian.net/browse/IR-1040 Add transport retrieval method for interfaces 
They both cover the issue with dependency injection